package.json

```diff
@@ -7,6 +7,7 @@
     "url": "https://github.com/cloudflare/node-cloudflare/issues"
   },
   "dependencies": {
+    "@sindresorhus/is": "^0.12.0",
```

Comment:
The two type checks can be done without importing this package.

Reply:
Thanks for the feedback.

I think I got everything incorporated, let me know if you think it should be done differently.